sw/device/tests/aon_timer_irq_test.c

```diff
@@ -57,8 +60,9 @@ static_assert(configUSE_PREEMPTION == 0,
               "rv_timer may be initialized already by FreeRtos");
 
 static void tick_init(void) {
+  dt_device_t rv_timer_dev = dt_get_device(kDtDeviceTypeRvTimer, 0);
```

Comment:
To see how LTO affects the output, you can look at the disassembly.

For this test (`//sw/device/tests:aon_timer_irq_test_fpga_cw310_test_rom`), the file to observe would be at `bazel-out/k8-fastbuild-ST-2cc462681f62/bin/sw/device/tests/aon_timer_irq_test_fpga_cw310_test_rom.dis`, for example. You would see that virtually all the devicetables functions (beginning with `dt_`) are inlined and reduced to the loading of constants.